Formula/gcc.rb

```diff
@@ -172,19 +174,7 @@ def install
     # Even when suffixes are appended, the info pages conflict when
     # install-info is run. TODO fix this.
     info.rmtree
-
-    # Rename java properties
-    if build.with?("java") || build.with?("all-languages")
-      config_files = [
-        "#{lib}/gcc/#{version_suffix}/logging.properties",
-        "#{lib}/gcc/#{version_suffix}/security/classpath.security",
-        "#{lib}/gcc/#{version_suffix}/i386/logging.properties",
-        "#{lib}/gcc/#{version_suffix}/i386/security/classpath.security",
-      ]
-      config_files.each do |file|
-        add_suffix file, version_suffix if File.exist? file
-      end
-    end
+    # Since GCC 4.9 java properties are properly sandboxed.
```

Comment:
@mikemcquaid It looks like they are installed in the same location: lib/gcc/6/ ... but the specialized code above is renaming them from e.g., `lib/gcc/6/logging.properties` to `lib/gcc/6/logging-6.properties`

```
$ find /usr/local/homebrew/Cellar/gcc/6.1.0 -name 'logging.properties'
/usr/local/homebrew/Cellar/gcc/6.1.0/lib/gcc/6/i386/logging.properties
/usr/local/homebrew/Cellar/gcc/6.1.0/lib/gcc/6/logging.properties
```

here is the add suffix function:

```ruby
  def add_suffix(file, suffix)
    dir = File.dirname(file)
    ext = File.extname(file)
    base = File.basename(file, ext)
    File.rename file, "#{dir}/#{base}-#{suffix}#{ext}"
  end
```

@manphiz do you know why this was done in gcc6.rb?